Sentinel-3 (#182)
microsoft/planetary-computer-tasks: datasets/sentinel-3/collection/sentinel-3-olci-lfr-l2-netcdf/description.md

This collection provides the Full Resolution [OLCI Level-2 Land][olci-l2] products containing data on global vegetation, chlorophyll, and water vapor.

## Data files

This dataset includes data on three primary variables:

* OLCI global vegetation index file
* terrestrial Chlorophyll index file
* integrated water vapor over water file.

> Each variable is contained within a separate NetCDF file, and is cataloged by an asset on each Item (`ogvi` for the vegetation index, )

Author: ```suggestion
Each variable is contained within a separate NetCDF file, and is cataloged as an asset in each item.
```

Reviewer: fixed.
